Issue #237: dashed syntax for line number highlights

Cthomp7 opened the pull request:
Added extra `if` statement so when `_getData` is generated it will accept dashed values (ex: `5-8`). Everything worked when downloading JSON & generating link + running example #237

nbriz commented on www/widgets/demo-example-maker/index.js:
```diff
@@ -297,10 +297,20 @@ class DemoExampleMaker extends Widget {
       if (s.focus && typeof s.focus === 'string') {
-        s.focus = s.focus.split(',').map(n => Number(n))
+        s.focus = s.focus.split(',').flatMap(f => {
+          console.log(f)
+          if (f.includes('-')) {
+            const [start, end] = f.split('-').map(n => Number(n))
+            console.log(Array.from({ length: end - start + 1 }, (_, i) => start + i))
```
don't forget to remove console.logs after everything'ss been tested

nbriz reviewed (changes requested):
i've tested with the share link && it works great! (still need to test json file, but i assume that works well too), only time it **doesn't work is when u click "preview"**, dashes get interpreted the same as commas.

side note: i'd never heard of [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) before! very efficient way to implement this :)

nbriz commented:
@Cthomp7 any progress on this one? TL;DR

- dashed lines don't work when clicking "preview" button
- need to remove console logs

Cthomp7 commented:
@nbriz Did and done! I cleaned up the two `console.log`s and added the same `flatMap` functionality to the preview function as well. Let me know if you have any other issues 🐸

nbriz commented:
perfect! thnx @Cthomp7 merging now!
